stock_outgoing_shipment_report/views/stock_outgoing_shipment_report_views.xml

```diff
@@ -6,6 +6,8 @@
         <field name="arch" type="xml">
             <search string="Outgoing Moves">
                 <field name="carrier_id" />
+                <filter name="createdtoday" string="Created Today" domain="[('create_date','&gt;=', datetime.datetime.combine(context_today(), datetime.time(0,0,0))), ('create_date','&lt;=', datetime.datetime.combine(context_today(), datetime.time(23,59,59)))]"/>
+                <filter name="date_created" string="Created Date (Date Only)" date="date_created" />
```

Comment:
```suggestion
                <filter name="date_created" string="Created Date" date="date_created" />
```
Non-blocking suggestion.  'Date Only' may just make them wonder what it means, and therefore I'd prefer to hide it here.

Reply:
Thank you, I understand.
After correspond, I'll merge this PR.